skbio/sequence/distance.py

```diff
@@ -113,3 +104,54 @@ def hamming(seq1, seq2):
         distance = scipy.spatial.distance.hamming(seq1.values, seq2.values)
 
     return float(distance)
+
+
+@experimental(as_of='0.4.2')
+def kmer_distance(seq1, seq2, k=3, overlap=True):
+    """Compute the kmer distance between a pair of sequences
+    Parameters
+    ----------
+    seq1 : skbio.Sequence
```

Comment:
Change parameters section to:

```
seq1, seq2 : Sequence
    Sequences to compute kmer distance between.
k : int
    The kmer length.
overlap : bool, optional
    Defines whether the kmers should be overlapping or not.
```

I made some minor changes to the text to be consistent with `hamming`, `Sequence.iter_kmers`, and `Sequence.kmer_frequencies`.